source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -25,16 +25,16 @@ This page provides an overview of MinIO's availability and resiliency design and
    Community Support is best-effort only and has no SLAs around responsiveness.
 
 MinIO implements :ref:`erasure coding <minio-erasure-coding>` as the core component in providing availability and resiliency during drive or node-level failure events.
-   MinIO shards each object into data and :ref:`parity <minio-ec-parity>` blocks and distributes those blocks across a single :ref:`erasure set <minio-ec-erasure-set>`.
+   MinIO partitions each object into data and :ref:`parity <minio-ec-parity>` shards and distributes those shards across a single :ref:`erasure set <minio-ec-erasure-set>`.
 
    .. figure:: /images/availability/availability-erasure-sharding.svg
       :figwidth: 100%
       :align: center
-      :alt: Diagram of erasure coded object sharded into 12 data blocks and 4 parity blocks
+      :alt: Diagram of erasure coded object partitioned into 12 (twelve) data shards and 4 (four) parity shards
```

Comment:
Very tiny nit: since the alt text is primarily for screen reader users, `12 (twelve)` is more likely to confuse than it would in the body text. (Where the need is different because overwhelmingly sighted readers.)

Reply:
Good catch - will fix throughout anywhere i did that.